docs/book/src/developer/providers/implementers-guide/configure.md

```diff
@@ -46,7 +46,7 @@ patchesStrategicMerge
 - manager_config.yaml
 ```
 
-[kustomizeyaml]: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#kustomization
+[kustomizeyaml]: https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#kustomization
```

Comment:
Wouldn't https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/ be better here?

Reply:
Thx, you're right. Done